packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -97,14 +102,14 @@ class FlutterCommandRunner extends CommandRunner<Null> {
             'Use this to select a specific version of the engine if you have built multiple engine targets.\n'
             'This path is relative to --local-engine-src-path/out.');
     argParser.addOption('record-to',
-        hide: !verboseHelp,
+        hide: true,
```

Comment:
Correct.  A script to create golden files will use it, and `--bug-report` wraps it, but users won't be directly exposed to it.